plugins/inputs/cpufreq/cpufreq.go

```diff
@@ -0,0 +1,120 @@
+package cpufreq
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const defaultHostSys = "/sys"
+
+type CPUFreq struct {
+	PathSysfs       string `toml:"host_sys"`
+	GatherThrottles bool   `toml:"gather_throttles"`
+}
+
+var sampleConfig = `
+  ## Path for sysfs filesystem.
+  ## See https://www.kernel.org/doc/Documentation/filesystems/sysfs.txt
+  ## Defaults:
+  # host_sys = "/sys"
+  ## Gather CPU throttles per core
+  ## Defaults:
+  # gather_throttles = false
+`
+
+func (g *CPUFreq) SampleConfig() string {
+	return sampleConfig
+}
+
+func (g *CPUFreq) Description() string {
+	return "Read specific statistics per cgroup"
+}
+
+func (g *CPUFreq) Gather(acc telegraf.Accumulator) error {
+
+	if g.PathSysfs == "" {
+		if os.Getenv("HOST_SYS") != "" {
+			g.PathSysfs = os.Getenv("HOST_SYS")
+		} else {
+			g.PathSysfs = defaultHostSys
+		}
+	}
+
+	cpus, err := filepath.Glob(path.Join(g.PathSysfs, "devices/system/cpu/cpu[0-9]*"))
+	if err != nil {
+		acc.AddError(err)
+	}
+
+	var value uint64
+	// cpu loop
+	for _, cpu := range cpus {
+		fileds := make(map[string]interface{})
```

Comment:
Typo
```suggestion
		fields := make(map[string]interface{})
```
also needs to be fixed below.